docs/index.yml

```
@@ -230,8 +232,10 @@ additionalContent:
             text: Windows Forms (.NET 5+)
           - url: /dotnet/desktop/winforms/?view=netframeworkdesktop-4.8&preserve-view=true
             text: Windows Forms (.NET Framework)
+          - url: /dotnet/maui
+            text: .NET Multi-platform App UI (.NET MAUI)            
```

This one has some spaces at the end
```suggestion
            text: .NET Multi-platform App UI (.NET MAUI)
```